packages/core/scripts/hardhat/verifyDeployedContracts.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+NETWORK_NAME=$1
+if [ ! "$NETWORK_NAME" ]; 
+then 
+    echo "Must specify network name"
+    exit 1
+fi
+
+# The network to verify Etherscan contracts for
+yarn hardhat etherscan-verify --license AGPL-3.0 --force-license --network $NETWORK_NAME
```

Comment:
Very cool!

Reply:
We need this actually because our license tag `SPDX-License-Identifier: AGPL-3.0-only` is "not supported" by Etherscan?

Reply:
Weird, I thought it was listed on etherscan when doing manual deployment....